config/__init__.py

```diff
@@ -1 +1,24 @@
-"""Holds the setings and entrypoints."""
+"""Define the configuration scheme using environ-config."""
+
+import logging
+
+import environ
+
+logger = logging.getLogger(__name__)
+
+
+@environ.config(prefix="SCRAM")
+class AppConfig:
+    """Top level configuration (i.e. SCRAM_)."""
+
+    debugger = environ.var(
+        default="", help='Will launch the appropriate debugger if set to either "pycharm-pydevd" or "debugpy".'
+    )
+
+    @debugger.validator
```

Comment:
I like this a lot more than having the debugger the way we did before. seems much cleaner